releases/release-7.5.1.md

```diff
@@ -0,0 +1,226 @@
+---
+title: TiDB 7.5.1 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 7.5.1.
+---
+
+# TiDB 7.5.1 Release Notes
+
+Release date: February 29, 2024
+
+TiDB version: 7.5.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v7.5/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v7.5.1#version-list)
+
+## Compatibility changes
+
```

Comment:
```suggestion

- Prohibit setting [`require_secure_transport`](https://docs.pingcap.com/tidb/v7.5/system-variables#require_secure_transport-new-in-v610) to `ON` in Security Enhanced Mode (SEM) to prevent potential connectivity issues for users [#47665](https://github.com/pingcap/tidb/issues/47665) @[tiancaiamao](https://github.com/tiancaiamao)
```